bin/docker-server

```diff
@@ -6,6 +6,8 @@ gunicorn posthog.wsgi \
     --bind 0.0.0.0:8000 \
     --log-file - \
     --log-level info \
+    --capture-output \
```

Comment:
Did we add those args to capture the `print` statements from stdout? If yes, I'm inclined to ask to move those `print` to use standard python logging so that we don't need to add these.

Reply:
No not so much print statements, but the backtrace format output, for when the backend has an error in e2e. At the moment the django app output I don't think will be visible anywhere. I imagine we could update to output the app logging to a file instead of stdout, but I prefer using stdout for this.

What do you think? Is there a better way to have the app logging be output when running the app with gunicorn?